packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -62,17 +63,17 @@ const getMedia = (userAgent: string) => {
 	let isMobile = true;
 	let isTablet = false;
 	// In development, parse user agent string to determine media by device
-	if (process.env.NODE_ENV !== 'production') {
-		const device = new MobileDetect(userAgent);
-		isMobile = Boolean(device.phone());
-		isTablet = Boolean(device.tablet());
-	} else {
+	if (process.env.NODE_ENV === 'production') {
```

Comment:
just a linting fix - `if/else` should start with positive (non-negated) condition

Reply:
maybe move the comment on line 65 down to the `else`?